spec/rspec/core/backtrace_formatter_spec.rb

```diff
@@ -112,19 +112,7 @@ def make_backtrace_formatter(exclusion_patterns=nil, inclusion_patterns=nil)
 
         expect(BacktraceFormatter.new.format_backtrace(backtrace)).to eq(["./my_spec.rb:5"])
       end
-
-      it "excludes lines from rspec libs by default", :failing_on_appveyor, :if => RSpec::Support::OS.windows? do
-        backtrace = [
-          "\\path\\to\\rspec-expectations\\lib\\rspec\\expectations\\foo.rb:37",
-          "\\path\\to\\rspec-expectations\\lib\\rspec\\matchers\\foo.rb:37",
-          ".\\my_spec.rb:5",
-          "\\path\\to\\rspec-mocks\\lib\\rspec\\mocks\\foo.rb:37",
-          "\\path\\to\\rspec-core\\lib\\rspec\\core\\foo.rb:37"
-        ]
-
-        expect(BacktraceFormatter.new.format_backtrace(backtrace)).to eq([".\\my_spec.rb:5"])
-      end
-
+      
```

Comment:
Looks like this got pulled into the diff from your merge of master into this branch?  When you asked before about pushing updates to this PR vs a new one I didn't realize this would happen.  FWIW, we tend to rebase and force push to PRs.   Having a PR composed of just its own commits (and not merges, etc) makes it easier to review.  Once we've got things in a state that's about ready to merge, would it be much trouble to clean up the history, either in this PR or via a new one (whatever is easier for you)?